common/device-model.cpp

```diff
@@ -27,22 +27,25 @@ using namespace rs2::sw_update;
 
 namespace rs2
 {
-    void imgui_easy_theming(ImFont*& font_14, ImFont*& font_18, ImFont*& monofont)
+    void imgui_easy_theming(ImFont*& font_14, ImFont*& font_18, ImFont*& monofont, int& font_size)
     {
         ImGuiStyle& style = ImGui::GetStyle();
 
         ImGuiIO& io = ImGui::GetIO();
         io.IniFilename = nullptr;
 
         const int OVERSAMPLE = config_file::instance().get(configurations::performance::font_oversample);
+        font_size = config_file::instance().get( configurations::window::font_size );
 
         static const ImWchar icons_ranges[] = { 0xf000, 0xf999, 0 }; // will not be copied by AddFont* so keep in scope.
 
         {
             ImFontConfig config_words;
             config_words.OversampleV = OVERSAMPLE;
             config_words.OversampleH = OVERSAMPLE;
-            font_14 = io.Fonts->AddFontFromMemoryCompressedTTF(karla_regular_compressed_data, karla_regular_compressed_size, 16.f);
+            font_14 = io.Fonts->AddFontFromMemoryCompressedTTF( karla_regular_compressed_data,
```

Comment:
May I change the variable name (font_14) to a new name because the name is not relevant?

Reply:
Yes, it will not be font 14 anymore.

Reply:
I will call it font_dynamic